pkgs/tools/security/vaultwarden/default.nix

```diff
@@ -37,6 +37,7 @@ rustPlatform.buildRustPackage rec {
   meta = with lib; {
     description = "Unofficial Bitwarden compatible server written in Rust";
     homepage = "https://github.com/dani-garcia/vaultwarden";
+    changelog = "https://github.com/dani-garcia/vaultwarden/releases/tag/${version}";
     license = licenses.agpl3Only;
     maintainers = with maintainers; [ msteen ivan ];
```

Comment:
Maybe I should replace msteen as they seem to be MIA since end of Mai last year.